Directory.Build.props

```
@@ -5,13 +5,28 @@
     <PackageProjectUrl>https://github.com/graphql-dotnet/server</PackageProjectUrl>
     <RepositoryUrl>https://github.com/graphql-dotnet/server</RepositoryUrl>
     <RepositoryType>Git</RepositoryType>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <Copyright>Pekka Heikura</Copyright>
     <Authors>Pekka Heikura</Authors>
     <Product>graphql-dotnet server</Product>
     <Company>graphql-dotnet</Company>
-    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <VersionPrefix>4.1.0-preview</VersionPrefix>
+    <VersionSuffix>dev</VersionSuffix>
+    <Deterministic>true</Deterministic>
+    <!-- https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables -->
+    <ContinuousIntegrationBuild Condition="'$(GITHUB_ACTIONS)' == 'true'">True</ContinuousIntegrationBuild>
+
+    <!-- Generate symbol packages (.snupkg) -->
+    <!-- You must publish both packages, the package that contains the dll (.nupkg) and the one that contains the symbols (.snupkg) -->
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

```suggestion
    <DebugType>embedded</DebugType>
    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

I am inclined to believe that it is easier and more convenient to make embedded symbols. I have watched a number of discussions about this in other projects.

---

👍

---

Also I would like to make sure we use the same options in other projects.